doc/reference/configuration/configuration_reference.rst

```diff
@@ -11,6 +11,329 @@ This topic describes all :ref:`configuration parameters <configuration>` provide
+
+        The option only has an effect if the :ref:`audit_log.to <configuration_reference_audit_to>` is set to ``syslog``
+        or ``pipe``.
+
+    |
+    | Type: boolean
+    | Default: false
+    | Environment variable: TT_AUDIT_LOG_NONBLOCK
+
+..  _configuration_reference_audit_pipe:
+
+..  confval:: audit_log.pipe
+
+    Specify a pipe for the audit log destination.
+    You can set the ``pipe`` type using the :ref:`audit_log.to <configuration_reference_audit_to>` option.
+    If log is a program, its pid is stored in the ``audit_log.logger_pid`` variable.
+    You need to send it a signal to rotate logs.
+
+    **Example**
+
+    ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/audit_log_pipe/config.yaml
+        :language: yaml
+        :start-at: audit_log:
+        :end-at: '| cronolog audit_tarantool.log'
+        :dedent:
+
+    This starts the `cronolog <https://linux.die.net/man/1/cronolog>`_ program when the server starts
+    and sends all ``audit_log`` messages to cronolog standard input (``stdin``).
+    If the ``audit_log`` string starts with '|',
+    the string is interpreted as a Unix `pipeline <https://en.wikipedia.org/wiki/Pipeline_%28Unix%29>`_.
+
+    |
+    | Type: string
+    | Default: box.NULL
+    | Environment variable: TT_AUDIT_LOG_PIPE
+
+..  _configuration_reference_audit_spaces:
+
+..  confval:: audit_log.spaces
+
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    The array of space names for which data operation events (``space_select``, ``space_insert``, ``space_replace``,
+    ``space_delete``) should be logged. The array accepts string values.
+    If set to :ref:`box.NULL <box-null>`, the data operation events are logged for all spaces.
+
+    **Example**
+
+    In the example, only the events of ``bands`` and ``singers`` spaces are logged:
+
+    ..  code-block:: yaml
+
+        audit_log:
+          spaces: [bands, singers]
+
+    |
+    | Type: array
+    | Default: box.NULL
+    | Environment variable: TT_AUDIT_LOG_SPACES
+
+..  _configuration_reference_audit_syslog-facility:
+
+..  confval:: audit_log.syslog_facility
+
+    Specify a system logger keyword that tells `syslogd <https://datatracker.ietf.org/doc/html/rfc5424>`__ where to send the message.
+    You can enable logging to a system logger using the :ref:`audit_log.to <configuration_reference_audit_to>` option.
+
+    See also: :ref:`syslog configuration example <configuration_reference_audit_syslog-example>`.
+
+    |
+    | Type: string
+    | Possible values: 'auth', 'authpriv', 'cron', 'daemon', 'ftp', 'kern', 'lpr', 'mail', 'news', 'security', 'syslog', 'user', 'uucp', 'local0', 'local1', 'local2', 'local3', 'local4', 'local5', 'local6', 'local7'
+    | Default: 'local7'
+    | Environment variable: TT_AUDIT_LOG_SYSLOG_FACILITY
+
+..  _configuration_reference_audit_syslog-identity:
+
+..  confval:: audit_log.syslog_identity
+
+    Specify an arbitrary string that will be placed at the beginning of all messages.
```

Comment:
> <...> string that will be placed at the beginning of all messages.

This is not exactly so.

I would better say what does it means: an application name to show in logs.

There is a brief description of the syslog format [here](https://github.com/tarantool/tarantool/blob/42d0a7a834ac75fb696d116379b00ce107c0ad1d/src/lib/core/say.c#L987-L999).

Reply:
Thanks, updated description